plugins/reporters/spdx/src/main/kotlin/SpdxDocumentReporter.kt

```diff
@@ -50,7 +53,13 @@ import org.ossreviewtoolkit.utils.spdx.model.SpdxDocument
  * - *file.information.enabled*: Toggle whether the output document should contain information on file granularity
  *                               about files containing findings.
  */
-class SpdxDocumentReporter : Reporter {
+@OrtPlugin(
+    displayName = "SPDX Document Reporter",
+    description = "Creates SPDX documents in YAML and JSON format.",
```

Comment:
I propose to align this a bit to the CycloneDX wording, which says "Creates software bills of materials (SBOM) in the CycloneDX format."

In particular, I would not mention the formats here (we also don't for CycloneDX, which support XML and JSON), but mention the term "SBOM" here as well.